packages/manager/apps/pci-instances/src/pages/instances/create/hooks/useCartIems.tsx

```diff
@@ -0,0 +1,158 @@
+import { NAMESPACES } from '@ovh-ux/manager-common-translations';
+import { Button, Text } from '@ovhcloud/ods-react';
+import { useTranslation } from 'react-i18next';
+import { TCartItem, TCartItemDetail } from '@/components/cart/Cart.component';
+import { FlavorDetails } from '../components/cart/FlavorDetails.component';
+import { useCatalogPrice } from '@ovh-ux/manager-react-components';
+import { useInstanceCreation } from './useInstanceCreation';
```

Comment:
Be careful here, if you couple this hook (I guess dedicated to handle only the cart) you will couple the UI component to your business / API logic so we could not move it anymore to a UI library